npm_and_yarn/helpers/build

```diff
@@ -8,13 +8,22 @@ if [ -z "$install_dir" ]; then
   exit 1
 fi
 
-helpers_dir="$(dirname "${BASH_SOURCE[0]}")"
+helpers_dir="$(dirname "$0")"
 cp -r \
-  "$helpers_dir/npm" \
-  "$helpers_dir/yarn" \
+  "$helpers_dir/lib" \
+  "$helpers_dir/test" \
+  "$helpers_dir/run.js" \
   "$helpers_dir/package.json" \
   "$helpers_dir/yarn.lock" \
   "$install_dir"
 
 cd "$install_dir"
-yarn install
+
+case "$DEPENDABOT_ENV" in
```

Comment:
Is this `DEPENDABOT_ENV` used somewhere? Think we should set `NODE_ENV=production` instead when running this script for the same effect.

Reply:
Oops good catch, actually for #986.